build.csx

```diff
@@ -92,4 +92,38 @@ targets.Add(
     }
 );
 
+targets.Add(
+    "run-gateway-demo",
+    DependsOn("find-msbuild", "restore-demos"),
```

Comment:
This line should be:

```C#
DependsOn("demos"),
```

I.e. you want to build the demos before running the API host exe's.

Reply:
thanks, I'll fix it